rotkehlchen/tests/api/test_compound.py

```diff
@@ -149,3 +152,37 @@ def test_events_compound(rotkehlchen_api_server: 'APIServer') -> None:
             },
         },
     }
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [[
+    '0x1107F797c1af4982b038Eb91260b3f9A90eecee9', '0x887380Bb5F5fF5C87BEcc46F0867Fec460F7c5a6',
+]])
+@pytest.mark.parametrize('have_decoders', [[True]])
+@pytest.mark.parametrize('ethereum_modules', [['compound']])
+def test_compound_v3_token_balances(
+        rotkehlchen_api_server: 'APIServer',
+        ethereum_accounts: 'list[ChecksumEvmAddress]',
+) -> None:
+    """Test that the balances of compound v3 supplied/borrowed tokens are correct."""
+    tokens = rotkehlchen_api_server.rest_api.rotkehlchen.chains_aggregator.ethereum.tokens
+    tokens.evm_inquirer.multicall = MagicMock(side_effect=tokens.evm_inquirer.multicall)  # type: ignore[method-assign]
+    tokens.detect_tokens(only_cache=False, addresses=[ethereum_accounts[1]])
+    rotkehlchen_api_server.rest_api.rotkehlchen.chains_aggregator.ethereum.transactions_decoder.decode_transaction_hashes(
+        ignore_cache=True,
+        tx_hashes=[deserialize_evm_tx_hash('0xe2229f1bdac86237e81ce96fcd694cf58b412dd1617746528735a319e0bb7595')],  # Borrow 0.1 WBTC from compound v3  # noqa: E501
+    )
+    response = requests.get(api_url_for(
+        rotkehlchen_api_server,
+        'blockchainbalancesresource',
+    ), json={'blockchain': 'ETH'})
+    result = assert_proper_response_with_result(response)['per_account']['eth']
```

Comment:
do you need the api server call here? Check how the others care called using the balance class itself and fetching the needed events